rs/pocket_ic_server/tests/test.rs

```diff
@@ -1478,3 +1482,88 @@ fn test_gateway_address_in_use() {
         port
     )));
 }
+
+fn test_canister_wasm() -> Vec<u8> {
+    let wasm_path = std::env::var_os("TEST_WASM").expect("Missing test canister wasm file");
+    std::fs::read(wasm_path).unwrap()
+}
+
+#[test]
+fn test_custom_blockmaker() {
```

Comment:
let's move this test to //packages/pocket-ic and extend the test canister there; please use candid::Principal instead of ic_types::PrincipalId and mgmt canister types from pocket_ic::management_canister_types